### PR TITLE
Fix issues affecting polymer-project.org site.

### PR DESCRIPTION
**Pull request by aomarks**
Found these issues by using Polymer 1.x, the 1.x markdown elements, and turning on HTML minification.

### CODE REVIEW

File: iron-doc-hide-bar.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
     <a href="" on-tap="_toggle">
       <span hidden$="[[visible]]">Show</span>
       <span hidden$="[[!visible]]">Hide</span>
-      <slot></slot>
+      &#8203;<slot></slot>
```
Comment (samuelli) [inline iron-doc-hide-bar.html:40]: why are these necessary?

**Review by samuelli — APPROVED** [review]

File: iron-doc-hide-bar.html (hunk 1 above)
Comment (aomarks) [reply iron-doc-hide-bar.html:40]: Whitespace was being collapsed by html-minifier (which is what polymer cli uses). Throwing in zero width spaces is a gross hack that forces it to not collapse as aggressively in these cases. I think maybe we should be using the `conservativeCollapse` option (https://www.npmjs.com/package/html-minifier#options-quick-reference), because it's pretty frustrating when the dev version renders differently to the built version.
